terasoluna-gfw-common-libraries/terasoluna-gfw-common/src/test/java/org/terasoluna/gfw/common/codelist/AbstractReloadableCodeListTest.java

```diff
@@ -15,24 +15,48 @@
  */
 package org.terasoluna.gfw.common.codelist;
 
-import static org.junit.Assert.assertThat;
 import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
 
 import java.util.HashMap;
 import java.util.Map;
 
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.terasoluna.gfw.common.codelist.AbstractReloadableCodeList;
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.Context;
 
 /**
  * Abstract class for the reloadable codelist functionality
  */
 public class AbstractReloadableCodeListTest {
 
+    private final String LOGBACK_UNIT_TEST_FILE_PATH = "src/test/resources/logback-unittest.xml";
+
+    private final String LOGBACK_DEFAULT_FILE_PATH = "src/test/resources/logback.xml";
+
+    private String logbackUnitTestFilePath = LOGBACK_UNIT_TEST_FILE_PATH;
+
+    private String logbackDefaultFilePath = LOGBACK_DEFAULT_FILE_PATH;
+
+    private Logger logger;
+
+    private Context context;
+
+    private JoranConfigurator configurator;

```

Comment:
Thx.
* https://github.com/terasolunaorg/terasoluna-gfw/commit/c77d96f054b5cf3a68bf691c32c61310c59274e7#diff-875c7873c28c403523aadb8ac4a6e62fR25
  Is `clearProperty` match content? ex. `resetLogLevel` etc...

* https://github.com/terasolunaorg/terasoluna-gfw/commit/c77d96f054b5cf3a68bf691c32c61310c59274e7#diff-875c7873c28c403523aadb8ac4a6e62fR33
  `configure` and `setting` are semantically overlap. You'd better change the metod name, ex  `reloadLogbackConfiguration` etc...